cmake/onnxruntime_python.cmake

```diff
@@ -96,7 +109,9 @@ if (onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS)
   target_link_libraries(onnxruntime_pybind11_state PRIVATE onnxruntime_language_interop onnxruntime_pyop)
 endif()
 
-
+if (onnxruntime_ENABLE_TRAINING_TORCH_INTEROP)
+  target_link_libraries(onnxruntime_pybind11_state PRIVATE onnxruntime_interop_torch)
```

Comment:
Move this section up
```cmake
if (onnxruntime_ENABLE_TRAINING_TORCH_INTEROP)
  target_link_libraries(onnxruntime_pybind11_state PRIVATE onnxruntime_interop_torch)
endif()
```

Like, if onnxruntime_interop_torch depends on onnxruntime_framework, then it should appear before onnxruntime_framework.

Reply:
ok

Reply:
It is because https://github.com/microsoft/onnxruntime/blob/master/docs/cmake_guideline.md#static-library-order-matters

Reply:
will update in new commit. thanks.